src/unix/core.c

```diff
@@ -1869,6 +1879,23 @@ int uv__search_path(const char* prog, char* buf, size_t* buflen) {
   return UV_EINVAL;
 }
 
+#if __NetBSD__
+int CPU_COUNT(cpuset_t *cpuset) {
```

Comment:
```suggestion
static int CPU_COUNT(cpuset_t *cpuset) {
```
And it'd be better to call it e.g. `uv__cpu_count`